app/demo-interactor/detail/DetectorUtils.cu

```diff
@@ -38,10 +38,11 @@ __global__ void bin_buffer_kernel(DetectorPointers const detector)
         // Add energy deposition (NOTE: very slow on arch 600)
         atomic_add(&detector.tally_deposition[bin],
                    hit.energy_deposited.value());
-        // detector.tally_deposition[bin] += hit.energy_deposited.value();
```

Comment:
Thanks -- this fell out of my rebase. I had it briefly as a performance test for the K40.